### PR TITLE
Dr. Ecco Changes

### PR DESCRIPTION
**Pull request by ZacharyATanenbaum**
Added an index.php file to allow Word-Finder to be added to the Dr. Ecco website. 

Also did AI stuff Joanna stole and implemented into the game without me. :(

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php $base = "../../" ?>
+    <base href="../../">
+    <script src="js/jquery-2.2.4.min.js"></script>
```
Comment (jbitton) [inline index.php:6]: why do we need jquery?

File: index.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    <script src="js/facebox.js"></script>
+    <script src="js/gameSettings.js"></script>
+    <link rel="stylesheet" type="text/css" href="css/facebox.css"/>
+    <link rel="stylesheet" type="text/css" href="css/main.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
```
Comment (jbitton) [inline index.php:11]: or any of these stylesheets? is this for the initial page?

File: index.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
```
Comment (jbitton) [inline index.php:13]: can you find out the point of this?

File: index.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    <?php include $base."header.php"; ?>
+    <nav>
+        <ul>
+        <li><a href="">Home</a></li>
+        </ul>
+        <?php include $base."leftMenuGame.php"; ?>
+    </nav>
+    <article>
+        <h1 id="gameName">Word-Finder</h1>
+        <h3 id="groupName">The Raging Dissapointments</h3>
```
Comment (jbitton) [inline index.php]: Disappointments

File: index.php [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        <h3>Instructions:</h3>
+        <div id="gameDesc" class="jumbotron">
+            <div>
+                <p>
+                    <b>How to Win</b>
+                    <br>
+                    The goal of the game is to have the most points when the game ends. The game ends when all rounds have been completed. A round being a cycle where each player gets a single turn.
```
Comment (jbitton) [inline index.php]: "The goal of the game is to accumulate the most points by the time the game finishes. The game has <code>n</code> rounds (defined by the player). A round consists of a turn for each player."

Comment (jbitton) [reply index.php]: the n is highlighted because I wrapped it in the `<code>` tag

File: index.php [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+                </p>
+                <p>
+                <br>
+                    <b>Turns</b>
+                    <br>
+                    A turn is split into two section - The Action Phase and Word Phase. In the Action Phase a player gets to rotate a column (up/down) a row (right/left) or swap a letter with any other letter. The Word Phase is where a player may select a word on the board to remove for points. The removed letters are then re-populated randomly.
```
Comment (jbitton) [inline index.php]: "A turn is split into two sections"

"In the Action Phase, a player gets a chance to either: rotate a column (up/down), a row (right/left), or swap a letter with any other letter"

"The Word Phase allows a player to select a word on the board for points, or to skip the selection. The selected letters are then re-populated randomly."

File: index.php [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+                </p>
+                <p>
+                <br>
+                    <b>Rules</b>
+                    <br>
+                <ul>
+                    <li>A word may only go from left to right or up to down.</li>
+                    <li>A word must be adjacent letters that are not diagonal.</li>
```
Comment (jbitton) [inline index.php]: "A word must be formed with adjacent letters that are not diagonal."

File: index.php [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+                    <li>The maximum board size is 12 letters by 12 letters.</li>
+                    <li>There may be up to four players and a bot. His name is Botty Mc. BotFace.</li>
```
Comment (jbitton) [inline index.php]: The name of the bot is "Bot"

File: index.php [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+                    <li>There may be up to four players and a bot. His name is Botty Mc. BotFace.</li> 
+                </ul>
+                </p>
+            </div>
+        </div>
+
+        <h3>Play It!</h3>
+        <form id="gameSettings" class="well">
+        </form>
+<!--        <iframe src="games/Word-Finder/build/index.html" class="game" width="800" height="800"></iframe>-->
```
Comment (jbitton) [inline index.php]: if you're commenting this line out, just remove it

**Review by jbitton — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (ZacharyATanenbaum) [reply index.php:6]: It's required for the "Play Now!" button on the bottom of the page.

File: index.php (hunk 2 above)
Comment (ZacharyATanenbaum) [reply index.php:11]: Yes, it's used for the initial page to have a similar style as the other Dr. Ecco pages.

File: index.php (hunk 3 above)
Comment (ZacharyATanenbaum) [reply index.php:13]: This script is used for creating the "Play Now!" button at the bottom of the page. The style sheets are used for making the page look similar to the other Dr. Ecco pages.

File: index.php (hunk 4 above)
Comment (ZacharyATanenbaum) [reply index.php]: I'm disappointed in myself.

File: index.php (hunk 5 above)
Comment (ZacharyATanenbaum) [reply index.php]: Added the changes to the file.

File: index.php (hunk 6 above)
Comment (ZacharyATanenbaum) [reply index.php]: Updated based on our conversation.

File: index.php (hunk 7 above)
Comment (ZacharyATanenbaum) [reply index.php]: Updated.

File: index.php (hunk 9 above)
Comment (ZacharyATanenbaum) [reply index.php]: Done.

**Pull request merged**